app/src/main/java/com/android/shelfLife/ui/newProfile/ProfileScreen.kt

```diff
@@ -80,13 +90,53 @@ fun ProfileScreen(
                 Image(
                     imageVector = Icons.Outlined.AccountCircle,
                     contentDescription = "Profile Picture",
-                    modifier = Modifier.size(100.dp).clip(CircleShape).testTag("profilePicture"),
+                    modifier =
+                        Modifier.size(100.dp)
+                            .clip(CircleShape)
+                            .testTag("profilePicture")
+                            .clickable {
+                              val currentTime = System.currentTimeMillis()
+                              if (currentTime - lastClickTime <= thresholdTime) {
+                                // Increment the click count for quick successive clicks
+                                clickCount++
+                              } else {
+                                // Reset the count if time exceeded threshold
+                                clickCount = 1
+                              }
+                              lastClickTime = currentTime
+
+                              // Check if the count has reached 5
+                              if (clickCount == 5) {
+                                navigationActions.navigateTo(Screen.EASTER_EGG)
+                                clickCount = 0 // Reset the counter after navigating
+                              }
+                            },
```

Comment:
### Changes to avoid duplication, and keep the MVVM architecture clean.
I understand that this functionality is an easter egg, but it might have been better to put this in the `ProfileScreenViewmodel` isntead, since this is some logic that could definitely be seperated from the UI.

Reply:
Also I noticed that you duplicated this code for the `AsyncImage`: this is  another reason to have this as a function in a ViewModel that you would call.